pyproject.toml

```diff
@@ -139,6 +139,7 @@ pikepdf = "^8.11.0"
 pypdf = "^4.0.1"
 python-docx = "^1.1.0"
 cohere = "5.0.0"
+django-allauth = {extras = ["saml"], version = "^0.61.1"}
```

Comment:
We already have django-allauth in `server.depedencies` group. Should we be modifying that?

Reply:
You are right - we should modify the existing `django-allauth`. Let me change this.